complex/fabtest.h

```diff
@@ -202,6 +209,7 @@ ssize_t ft_get_event(uint32_t *event, void *buf, size_t len,
 		     uint32_t event_check, size_t len_check);
 int ft_open_comp();
 int ft_bind_comp(struct fid_ep *ep, uint64_t flags);
+int ft_comp_x(struct fid_cq *cq, struct ft_xcontrol *ft_x, char *x_str, int nowait);
```

Comment:
I would keep ft_comp_x as a static call internal to ft_comp.c, and let the external users call the base ft_comp_rx/tx calls only.  This will simplify changes when we introduce counters.